src/containers/Review/ReviewPage.tsx

```diff
@@ -68,19 +70,30 @@ const ReviewPage: React.FC<{
 
   const { ConfirmModal: WarningModal, showModal: showWarningModal } = useConfirmationModal({
     type: 'warning',
-    title: strings.REVIEW_STATUS_PENDING_TITLE,
-    message: strings.REVIEW_STATUS_PENDING_MESSAGE,
+    onConfirm: () => push(`/application/${serial}/review`),
+    showCancel: false,
   })
 
   useEffect(() => {
-    if (!reviewStructure) return
-    if (thisReview?.current.reviewStatus === ReviewStatus.Pending) {
+    if (reviewStructure?.thisReview?.current.reviewStatus) {
+      console.log(reviewStructure.thisReview.current.reviewStatus)
+      setReviewStatus(reviewStructure.thisReview.current.reviewStatus)
+    }
```

Comment:
If you remove the `useState` variable (previous comment), then you won't need this either.